app/src/util/validation.ts

```diff
@@ -238,3 +238,10 @@ export const fetchEmissionsFactorRequest = z.object({
   regionLocode: z.string().optional(),
   metadata: z.record(z.string(), z.any()).optional(),
 });
+
+export const updatePasswordRequest = z.object({
+  currentPassword: z.string().min(4),
```

Comment:
Max length might be helpful as well since we're hashing it and the complexity there depends on the input length?